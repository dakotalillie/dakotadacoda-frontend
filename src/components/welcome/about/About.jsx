import React from 'react';
import { Col, Container, Row } from 'reactstrap';

import './about.css';
import profileImage from '../../../img/profile.jpg';

// Must remain a class-based component to work with scroll functionality.
export default class AboutPreview extends React.Component {
  render() {
    return (
      <div className="about section">
        <Container>
          <Row>
            <Col sm="9" className="copy_col">
              <h2>My name is Dakota</h2>
              <h2>I write code and make music</h2>
              <h5>(The two are not so dissimilar)</h5>
            </Col>
            <Col sm="3" className="profile_col">
              <img src={profileImage} className="profile_image" alt="profile" />
            </Col>
          </Row>
        </Container>
      </div>
    );
  }
}
