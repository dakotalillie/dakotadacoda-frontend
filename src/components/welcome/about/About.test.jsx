import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import About from './About';

describe('About', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<About />);
  });

  it('renders successfully', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});