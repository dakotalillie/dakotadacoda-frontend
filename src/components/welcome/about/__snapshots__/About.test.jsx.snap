// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders successfully 1`] = `
<div
  className="about section"
>
  <Container
    tag="div"
  >
    <Row
      tag="div"
    >
      <Col
        className="copy_col"
        sm="9"
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <h2>
          My name is Dakota
        </h2>
        <h2>
          I write code and make music
        </h2>
        <h5>
          (The two are not so dissimilar)
        </h5>
      </Col>
      <Col
        className="profile_col"
        sm="3"
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <img
          alt="profile"
          className="profile_image"
          src="profile.jpg"
        />
      </Col>
    </Row>
  </Container>
</div>
`;
